client/src/pages/Home.css

```diff
@@ -9,3 +9,46 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(214, 227, 243);
+  align-items: center;
+  height: 100%;
+  border-radius: 12px;
+  padding: 100px 100px 100px 100px;
+  width: 70%;
+  margin-left: 10em;
+  box-shadow: 3px 3px rgb(143, 139, 139), -1em 0 .4em rgb(211, 211, 198);
+  
+}
+.container-image{
+  
+  background-color: aliceblue;
+  border-radius: 150px;
+  border-style: solid;
+  
+  
+  
+}
+.button{
+  padding-top: 6px;
+
+
+}
+.signup{
+  padding-top: 10px;
+}
+.img{
+  
+  text-align: center;
+  
+}
```

Comment:
If we apply my suggestions above , we can delete these `button`, `signup`, and `img` classes as they're no longer being used